mingw-w64-freeglut/PKGBUILD

```
@@ -6,20 +6,21 @@ _realname=freeglut
 pkgbase="mingw-w64-${_realname}"
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 _pre=
-_ver=3.2.1
+_ver=3.2.2
 pkgver=${_ver}${_pre}
-pkgrel=2
+pkgrel=1
 pkgdesc="Freeglut allows the user to create and manage windows containing OpenGL contexts (mingw32-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
```

```suggestion
mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
```

This can be built for AArch64 now.